tests/acceptance/test_update.py

```diff
@@ -112,14 +114,17 @@ def install_update(image, conn, http_server, board_type, use_s3, s3_address):
             Helpers.upload_to_s3(image)
             http_server_location = "{}/mender/temp".format(s3_address)
         else:
-            http_server = subprocess.Popen(["python", "-m", "SimpleHTTPServer"])
+            http_server = subprocess.Popen(["python3", "-m", "http.server"])
             assert(http_server)
+            time.sleep(1)
+            assert requests.head("http://localhost:8000").status_code == 200
 
         try:
             output = conn.run("mender %s http://%s/%s" % (install_flag, http_server_location, image))
             print("output from rootfs update: ", output.stdout)
         finally:
             if http_server:
+                assert requests.head("http://localhost:8000").status_code == 200
```

Comment:
Why assert here? It's not very good style to have it in a cleanup function.

Reply:
Yes, that is not nice at all. But somewhat I want to check that the server is still up and running. I will add a simple print instead

Reply:
I reworked it and pushed. I will let all tests to run again to see if by a chance the issue appears.